provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/client/PpmsClient.java

```diff
@@ -1,34 +1,32 @@
 package gov.va.api.health.providerdirectory.service.client;
 
-import gov.va.api.health.providerdirectory.service.PpmsProviderSpecialtiesResponse;
-import gov.va.api.health.providerdirectory.service.ProviderContacts;
+import gov.va.api.health.providerdirectory.service.ProviderSpecialtiesResponse;
+import gov.va.api.health.providerdirectory.service.ProviderContactsResponse;
 import gov.va.api.health.providerdirectory.service.ProviderResponse;
 
 /**
- * This is the abstraction for communicating with the Mr. Anderson service. This service works with
- * a JAXB model that represents the CDW schemas. Queries to Mr. Anderson are contained in type-safe
- * objects.
+ * This is the abstraction for communicating with PPMS through their Restful API.
  */
 public interface PpmsClient {
   /** Return the parameters of the failed search. */
-  ProviderContacts providerContactsForId(String id);
+  ProviderContactsResponse providerContactsForId(String id);
 
   /** Return the parameters of the failed search. */
-  PpmsProviderSpecialtiesResponse providerSpecialtySearch(String id);
+  ProviderSpecialtiesResponse providerSpecialtySearch(String id);
 
   /** Return the parameters of the failed search. */
   ProviderResponse providersForId(String id);
 
   ProviderResponse providersForName(String id);
 
-  /** A request to Mr. Anderson was malformed, such as missing required search parameters. */
+  /** A request was malformed, such as missing required search parameters. */
   class BadRequest extends PpmsServiceException {
     public BadRequest(String id) {
       super(id);
     }
   }
 
-  /** The generic exception for working with Mr. Anderson. */
+  /** The generic exception for working. */
   class PpmsServiceException extends RuntimeException {
```

Comment:
`PpmsServiceException` can be shortened to `PpmsException`. And alphabetize the list of exceptions.